lib/cron_parser.rb

```diff
@@ -295,4 +304,16 @@ def validate_source
       raise ArgumentError, 'not a valid cronline'
     end
   end
+
+  def strict_match?
+    @strict_match
+  end
+
+  def set_strict_match(strict_match = false)
+    @strict_match = false
+    if strict_match && [time_specs[:dom].last, time_specs[:dow].last].all?{|v| v != '*'}
```

Comment:
Actually yes; when I did not have this check in place, other specs failed that should not have. I'll admit I only 60-70% get whats actually going on with the code, so maybe you'd know better than I why. or, maybe it was from something else I later undid. I'll see what happens when i remove it again.